### ISSUE TITLE
Issue with recast and unreachable

### ISSUE THREAD

**Issue by ghost** [zertosh/unreachable-branch-transform#14]
I'm not exactly sure the root cause of this issue, but I'm getting:

```
AssertionError: no line 0 (line numbers start from 1)
    at Lines.Lp.getIndentAt 
```

Which also happens when I do a simple

``` bash
$: node
$: var r = require('recast')
$: r.parse('')
```

There was a fix added to recast to address [this issue](https://github.com/benjamn/recast/issues/220) here https://github.com/benjamn/recast/commit/34b660650d23f05f2bdcc36f31c6851c7be60123, but the fix requires `0.11.4`. Can we update the package.json here to either use a peerDependency or just update the dependency directly?

(note: manually patching this library inside my node_modules with `0.11.4` makes the issue disappear).

**Comment by zertosh** [comment]
Sorry everyone. I'll be on a computer in a few hours and push a fix to unreachable-branch-transform.

**Comment by zertosh** [comment]
Published unreachable-branch-transform@0.5.1, which uses recast@^0.11.4.
